composer.json

```diff
@@ -31,7 +31,7 @@
     "minimum-stability": "dev",
     "extra": {
         "branch-alias": {
-            "dev-master": "1.0.0-dev"
+            "dev-master": "1.0.x-dev"
```

Comment:
This does not look like the correct change to make, and there is no 0.12.0 designation to be found in the composer file.

Reply:
I found this is correct; it allows someone to require thrift version 1.0.* and it will pull the master branch from the project, instead of a tagged version.

https://getcomposer.org/doc/articles/aliases.md#branch-alias